user/migrate/open-source-repository-migration.md

```diff
@@ -185,3 +185,49 @@ If you have any further questions, comments or need help on our Beta migration p
 
 [travis-ci.com]: https://www.travis-ci.com
 [travis-ci.org]: https://www.travis-ci.org
+
+## Frequently asked questions
+
+#### Q. When will the migration from travis-ci.org to travis-ci.com be completed?
+
+A. In an effort to ensure that all of our users - whether you build open-source, public or private repositories - receive regular feature updates, security patches and UX/UI enhancements, we are announcing that travis-ci.org will be officially closed down completely no later than December 31st, 2020, allowing us to focus all our efforts on bringing new features and fixes to travis-ci.com and all of our awesome users like yourself on the travis-ci.com domain.
+
+
+#### Q. What will happen to travis-ci.org after December 31st, 2020?
+
+A. Travis-ci.org will be switched to a read-only platform, allowing you to see your jobs build history from all repositories previously connected to travis-ci.org.
+
+
+#### Q. Why are some of my queued jobs taking longer than usual to build?
+
+A. At peak usage times, you may see your build times are longer than they previously have been - we need to make sure all users have equal access to the .org platform until the end of the year as we move our infrastructure across to .com. You may want to consider migrating across to .com sooner rather than later, or consider scheduling builds at a quieter time of the day if remaining on .org for a while longer.
+
+
+#### Q. Why are some of my queued jobs being cancelled?
+
+A. This might happen as we move infrastructure over from .org to .com - if we see a job stuck in a queue for more than 16 hours, we will review this on a case by case basis and possibly cancel the job to ensure resources are not being reduced across the platform. You may want to consider migrating across to .com sooner rather than later, or consider building at a quieter time of the day if remaining on .org for a while longer.
```

Comment:
This is due to the infrastructure migration from .org to .com - if a job is stuck in a queue for more than 16 hours, we will review it in a case by case basis and probably cancel the job to ensure resources are not reduced across the platform. You should consider migrating to .com sooner rather than later, or schedule your builds at a quieter time of the day.